packages/blob-storage-manager/src/storages/GoogleStorage.ts

```diff
@@ -5,21 +5,21 @@ import type { BlobStorageConfig } from "../BlobStorage";
 import { BlobStorage } from "../BlobStorage";
 import type { Environment } from "../env";
 
-interface GoogleStorageConfig extends BlobStorageConfig {
+export interface GoogleStorageConfig extends BlobStorageConfig {
   serviceKey?: string;
   projectId?: string;
   bucketName: string;
   apiEndpoint?: string;
 }
 
-type GoogleCredentials = {
+export type GoogleCredentials = {
   client_email: string;
   private_key: string;
 };
 
 export class GoogleStorage extends BlobStorage {
-  #storageClient: Storage;
-  #bucketName: string;
+  _storageClient: Storage;
```

Comment:
@PJColombo what do you think of make this protected instead to access them in the mock subclass?